compliance_checker/suite.py

```diff
@@ -425,8 +437,8 @@ def process_table(res, check):
                 # datasets
                 if _top_level:
                     print("\n")
-                    print('{:^80}'.format(level_name))
-                    print("-" * 80)
+                    print('{:^120}'.format(level_name))
```

Comment:
Should this be replaced by ```_len_ ``` like the others? Interesting variable name btw, i would've went with ```width```